tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -1542,27 +1497,17 @@ pub fn std_multi_bit_blind_rotate_assign<Scalar, InputCont, OutputCont, KeyCont>
 
 /// Deterministic variant of [`std_multi_bit_blind_rotate_assign`]. Performance
 /// may be slightly worse than the non deterministic version.
-pub fn std_multi_bit_deterministic_blind_rotate_assign<Scalar, InputCont, OutputCont, KeyCont>(
-    input: &LweCiphertext<InputCont>,
+pub fn std_multi_bit_deterministic_blind_rotate_assign<Scalar, OutputCont, KeyCont>(
+    switched_modulus_input: &impl MultiBitModulusSwitchedCt,
     accumulator: &mut GlweCiphertext<OutputCont>,
     multi_bit_bsk: &LweMultiBitBootstrapKey<KeyCont>,
     thread_count: ThreadCount,
 ) where
     // CastInto required for PBS modulus switch which returns a usize
     Scalar: UnsignedTorus + CastInto<usize> + CastFrom<usize> + Sync,
-    InputCont: Container<Element = Scalar> + Sync,
     OutputCont: ContainerMut<Element = Scalar>,
     KeyCont: Container<Element = Scalar> + Sync,
 {
-    assert_eq!(
```

Comment:
why remove ?

Reply:
I guess we could add a `lwe_dimension` method to the `MultiBitModulusSwitchedCt` trait 
Wdyt?

Reply:
sounds good

Reply:
Added